web/frontend/src/components/remote-control-cards.vue

```diff
@@ -54,6 +56,8 @@ import {
   fixServoStatus,
 } from '../lib/fixers';
 
+const AudioInput = svelteAdapter(AudioInputSvelte);
```

Comment:
This is pretty much all that's required to bridge the two frameworks. It bridges props, events, updates, and watchers. It probably breaks down if one were to utilize super obscure framework features in either Svelte or Vue, but we don't, and should be sufficient for a migration period.